acceptance-tests/.scalafmt.conf

```diff
@@ -19,7 +19,7 @@ align.openParenCallSite = true
 align.openParenDefnSite = true
 
 rewrite {
-  rules = [RedundantBraces, RedundantParens, AsciiSortImports, SortModifiers, PreferCurlyFors]
+  rules = [RedundantBraces, RedundantParens, SortModifiers, PreferCurlyFors]
```

Comment:
There is a scalafix semantic rule [organize imports](https://github.com/liancheng/scalafix-organize-imports) that I found quite nice to use. Perhaps something to look at/discuss for later.

Reply:
Yes, sure. I'm more than happy to talk about that. I simply removed it as I quite often organize imports with Intellij and I couldn't force it to use Ascii Sorting. There are few other options to choose from so maybe we can find a common denominator to satisfy both.